source/_data/themes.yml

```
@@ -1,3 +1,15 @@
+- name: LiveMyLife
+  description: A simple and pure personal Theme for Hexo...
+  link: https://github.com/V-Vincen/hexo-theme-livemylife
+  preview: https://v-vincen.life/
+  tags:
+    - simple
+    - pure
+    - clean
+    - dark/light
+    - personal
+    - practical
+
```

```suggestion
    - practical
```

No extra blank line needed.

---

Thanks I got it.